test/e2e/scaffold/ingress.go

```diff
@@ -256,6 +264,7 @@ spec:
             - %s
             - --apisix-route-version
             - %s
+            - --watch-endpointslices
```

Comment:
The default value of `watch-endpointslices` is false?

Reply:
The minimal Kubernetes version that we supporting is `v1.15` while the alpha version of endpointslice was introduced in Kubernetes `v1.16`. So by default we disable it.